emails/2023-11-09.md

```diff
@@ -0,0 +1,33 @@
+This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
+
+Many of you might not know that you can ask [Bing Chat](https://www.bing.com/search?form=MY0291&OCID=MY0291&q=Bing+AI&showconv=1) to draw you stuff. Yes, you could do this before with other services but this is free and accessible to most people now. So, [Zach](https://www.doltub.com/team#zach) used it to generate pictures for his [virtual columns blog](https://www.dolthub.com/blog/2023-11-03-virtual-columns/). When he shared the pictures, I asked it to draw Zach with this prompt:
+
+> draw me a picture of a red bearded man with a mullet in deep awe of what is happening on his computer screen
+
+The likeness is uncanny. Also, notice the mouse dual-wield.
+
+![Zach](../images/zach-dall-e.png)
+
+### DoltLab Enterprise
+
+Since it's launch in early 2022, [DoltLab](https://docs.dolthub.com/products/doltlab), our on-prem version of DoltHub, has slowly been making it's way into companies around the world. These companies have asked for features like look-and-feel customization and single sign on. We [launched DoltLab Enterprise](https://www.dolthub.com/blog/2023-10-30-announcing-doltlab-enterprise/) to satisfy those requests. It costs $5,000/month for unlimited users and you also get our great enterprise support bundled in.
```

Comment:
```suggestion
Since its launch in early 2022, [DoltLab](https://docs.dolthub.com/products/doltlab), our on-prem version of DoltHub, has slowly been making its way into companies around the world. These companies have asked for features like look-and-feel customization and single sign on. We [launched DoltLab Enterprise](https://www.dolthub.com/blog/2023-10-30-announcing-doltlab-enterprise/) to satisfy those requests. It costs $5,000/month for unlimited users and you also get our great enterprise support bundled in.
```